src/components/side-panel/layer-panel/layer-config-group.js

```diff
@@ -27,20 +27,33 @@ import Switch from 'components/common/switch';
 import InfoHelper from 'components/common/info-helper';
 import {VertThreeDots} from 'components/common/icons';
 
+const DashedPrependCss = `
+  div {
+    width: 12px;
+    border-top: 1px dashed #333c54;
+    margin-left: -17px;
+  }
+`;
 export const StyledLayerConfigGroupLabel = styled.div`
-  border-left: 2px solid ${props => props.theme.labelColor};
+  border-left: ${props => props.theme.styledLayerConfigGroupLabelBorderLeft} solid
+    ${props => props.theme.labelColor};
   line-height: 12px;
-  margin-left: -12px;
-  padding-left: 10px;
+  margin-left: ${props => props.theme.styledLayerConfigGroupLabelMargin};
```

Comment:
you can just call thses`layerConfigGroupLabelMargin`

Reply:
removed